package.json

```diff
@@ -43,19 +45,21 @@
     "next-auth": "^4.24.10",
     "next-seo": "^5.14.1",
     "papaparse": "^5.3.2",
+    "pino": "^9.5.0",
+    "pino-pretty": "^13.0.0",
     "react": "18.2.0",
     "react-cmdk": "^1.3.7",
     "react-dom": "18.2.0",
-    "react-hook-form": "^7.39.4",
+    "react-hook-form": "7.54.0",
     "react-icons": "^4.7.1",
     "react-infinite-scroller": "^1.2.6",
     "regenerator-runtime": "0.13.7",
     "sharp": "^0.33.0",
     "siwe": "2.3.2",
     "swr": "^2.2.4",
     "tslib": "^2.3.0",
-    "viem": "2.21.45",
-    "wagmi": "2.12.32",
+    "viem": "2.17.0",
```

Comment:
why are we decreasing the version of viem here?

Reply:
@dan13ram  to match smart invoice versions